src/main/java/com/travel/pay/facade/TravelTourFacade.java

```diff
@@ -0,0 +1,109 @@
+package com.travel.pay.facade;
+
+import com.travel.pay.model.Hotel;
+import com.travel.pay.model.Tour;
+import com.travel.pay.service.api.PaymentService;
+import com.travel.pay.service.api.TourService;
+import com.travel.pay.service.impl.PaymentServiceImpl;
+import com.travel.pay.service.impl.TourServiceImpl;
+import com.travel.pay.utils.DateTimeUtils;
+import one.util.streamex.StreamEx;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class TravelTourFacade {
+    private final PaymentService paymentService;
+    private final List<Tour> currentToursList;
+    private final List<Tour> intermediateToursList = new ArrayList<>();
+    private Tour currentTour;
+
+    public TravelTourFacade() {
+        this.paymentService = new PaymentServiceImpl();
+        TourService tourService = new TourServiceImpl();
```

Comment:
if you need a `TourService` just for internal collection retrieval, you can inline new instance creation with a getter call